.demo/env/charts/gatekeeper.yaml

```diff
@@ -0,0 +1,75 @@
+charts:
+    gatekeeper:
+        enabled: false
+        constraints:
+            - parameters:
+                enabled: false
+                cpu: 10m
+                memory: 120Mi
+              policyName: containerlimits
+            - parameters:
+                enabled: true
+                tags:
+                    - ''
+                    # - 0.3.7-debian-10-r103
+                    # - 5.0.7-debian-10-r32
+                    # - v1.1.0
+                    # - latest
+                    # - master
+              policyName: bannedimagetags
+            - parameters:
```

Comment:
why is this configurable here? we wan't to protect what is in the core and you expose it to be broken...

Reply:
anyway, this policy will also not be used as again it goes against a good workflow. Please start thinking about the implications of using such policies. Who will add to this list? Admins? But who needs to use images all the time? Devs. Not workable.

Again, this problem space is better served by something like harbor.

Reply:
@Morriz I recommend to start form defining spec for gatekeeper in values-schema.yaml file.
If we make a clear interface for values then there won't be any confusions about what is configurable and what is not.